docs/recipes/InjectingDependenciesIntoEpics.md

```diff
@@ -1,55 +1,79 @@
 # Injecting Dependencies Into Epics
 
-Sometimes you might want to have some dependencies injected into your epics. Things that you might want to mock in tests fall into this category.
+Injecting your dependencies into your Epics can help with testing.
 
-Let's say you want to interact with the network. You could use the `ajax` method directly from `rxjs`:
+Let's say you want to interact with the network. You could use the `ajax` helpers directly from `rxjs`:
 
 ```js
-import { ajax } from 'rxjs/observable/dom/ajax'
-
-export const fetchUsersEpic = action$ => action$.ofType('FETCH_USERS_REQUESTED')
-  .flatMap(() => ajax.getJSON('https://my.api.com/users'))
-  .map(response => ({ type: 'FETCH_USERS_COMPLETED', payload: response }))
+import { ajax } from 'rxjs/observable/dom/ajax';
+
+const fetchUserEpic = (action$, store) =>
+  action$.ofType('FETCH_USER')
+    .mergeMap(({ payload }) =>
+      ajax.getJSON(`/api/users/${payload}`)
+        .map(response => ({
+          type: 'FETCH_USER_FULFILLED',
+          payload: response
+        }))
+    );
```

Comment:
I mostly changed this to fetching a single user (simpler as an example to grok) and adjusted some things to be consistent with the other doc examples like mergeMap vs flatMap, isolating the observable chains by putting the map inside the mergeMap.